java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/WorkspaceServiceImpl.java

```diff
@@ -774,6 +776,25 @@ public CompletableFuture<Object> executeCommand(ExecuteCommandParams params) {
                     }
                     return (CompletableFuture<Object>)(CompletableFuture)((TextDocumentServiceImpl)server.getTextDocumentService()).computeDiagnostics(source, s);
             }
+            case Server.NBLS_GO_TO_TEST: {
```

Comment:
Given we have a `CommandProvider` now, consider to use that to keep `WorkspaceServiceImpl` from bloating even more.